extensions/banner/deployment/src/main/java/io/quarkus/banner/deployment/BannerConfig.java

```diff
@@ -5,6 +5,13 @@
 
 @ConfigRoot(name = "banner")
 public class BannerConfig {
+
+    /**
+     * Whether or not the banner will be displayed
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean enabled;
```

Comment:
I would have made it runtime so I would read the banner in any case and just condition the display at runtime.

Reply:
I on the other hand love it😎. To make it makes things very clear. Plus we use it all over the place.

Indeed if we decide to make the property a runtime one it will have to change

Reply:
I would say let's keep it build time for now. We can always make it runtime in the future if needed